fastrand.go

```diff
@@ -111,6 +108,9 @@ func Intn(n int) int {
 }
 
 // BigIntn returns a uniform random value in [0,n). It panics if n <= 0.
+//
+// TODO: This particular function has not been extended to be faster, as it is
+// just a passthrough.
```

Comment:
The `BenchmarkBigIntn` test uses a 32-byte big.Int, so its performance can be compared with `BenchmarkRead32` (which takes ~200ns). Profiling, it looks like the main reason `BigIntn` is slower is that it has to allocate the big.Int on the heap. If we were reading into a preallocated big.Int I imagine the benchmark would drop to nearly 200ns. Not worth the more annoying API, though.